ivy/stateful/layers.py

```diff
@@ -1983,3 +1983,66 @@ def _forward(self, inputs):
             self._padding,
             data_format=self._data_format,
         )
+
+
+class Dct(Module):
+    def __init__(
+        self,
+        *,
+        type=2,
+        n=None,
+        axis=-1,
+        norm=None,
+        out=None,
```

Comment:
We don't need this 'out' kwarg in stateful layers. Please can you remove it, its description in the docstring, etc?